pom.xml

```diff
@@ -0,0 +1,117 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.wso2.carbon</groupId>
+    <artifactId>mongodb.user.store.mgt</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <packaging>bundle</packaging>
+
+    <repositories>
+        <repository>
+            <id>wso2-nexus</id>
+            <name>WSO2 internal Repository</name>
+            <url>http://maven.wso2.org/nexus/content/groups/wso2-public/</url>
+            <releases>
+                <enabled>true</enabled>
+                <updatePolicy>daily</updatePolicy>
+                <checksumPolicy>ignore</checksumPolicy>
+            </releases>
+        </repository>
+    </repositories>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.user.core</artifactId>
+            <version>4.4.26</version>
```

Comment:
as a best practice create  <properties> section and put versions.

    <properties>
        <project.scm.id>scm-server</project.scm.id>

        <!-- File encoding of build -->
        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>

        <!-- APIM Component Version -->
        <hamcrest.version>1.3</hamcrest.version>
        <carbon.apimgt.version>6.2.263-SNAPSHOT</carbon.apimgt.version>
        <carbon.apimgt.imp.pkg.version>[6.0.0, 7.0.0)</carbon.apimgt.imp.pkg.version>
----

Reply:
Ack. I have moved the versions to properties section now. Also upgraded the dependencies to latest versions.